.travis/install_test.sh

```diff
@@ -4,7 +4,12 @@ set -ev
 mkdir --parents "${HOME}/bin"
 
 # PHPUnit install
-wget "https://phar.phpunit.de/phpunit-7.phar" --output-document="${HOME}/bin/phpunit"
```

Comment:
@greg0ire this one is now rewritten.
Do you find some time to make phpunit version configurable on dev-kit?

Reply:
ping :)

Reply:
Woops no I didn't find the time, I remember a solution that involves searching for php 5 in the array though